policySet.yml

```diff
@@ -0,0 +1,36 @@
+apiVersion: policy.open-cluster-management.io/v1beta1
```

Comment:
Do you plan on actually including this file and policyset.yml ?
If yes - why?

Reply:
No, that's probably a leftover i missed, deleted, thanks.